satpy/etc/readers/ghrsst_l2.yaml

```diff
@@ -0,0 +1,97 @@
+reader:
+  description: NC Reader for GHRSST Level 2 data
+  name: ghrsst_l2
```

Comment:
The main reader table on the first satpy documentation page needs to be update for this name change.

Reply:
Oh, yes, forgot that, thanks for catching this!